spring-cloud-kubernetes-client-autoconfig/src/main/java/org/springframework/cloud/kubernetes/client/KubernetesClientAutoConfiguration.java

```diff
@@ -46,9 +47,11 @@
 
 	@Bean
 	@ConditionalOnMissingBean
-	public ApiClient apiClient() throws IOException {
+	public ApiClient apiClient(@Value("${spring.application.name:application}") String applicationName) {
```

Comment:
@ryanjbaxter when i moved `@Value` annotation injection to `KubernetesClientProperties` the integration test becomes constantly failing so i reverted to injection via parameters. i can't reproduce that locally b/c all the integration tests are passing perfectly. wdyt?

Reply:
We need to figure out why they are failing.  We should not need to do this via parameter injection.

Reply:
is there any way to browse the log from the integration-testing container? by adding some extra printing lines, i found that in the failing tests, the `/actuator/info` endpoint is returning `{"inside":false}` which results the failure --- while actually it's running inside a cluster.

Reply:
Not 100% sure, maybe just reproduce outside of the integration test in your own sample?

Reply:
oddly, everytime i ran the integration tests locally, they pass perfectly... let me figure out what's wrong the in the remote test..

Reply:
@yue9944882 if you do not mind, I will be taking this one where you left and bring it to a PR ready to review state.